tests/functests/APITest.cpp

```diff
@@ -290,6 +290,29 @@ TEST(APITest, LogManager_Initialize_Custom)
     LogManager::FlushAndTeardown();
 }
 
+#define SOME_CUSTOM_URL "https://custom.url"
```

Comment:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-macros2

Reply:
+1. the preprocessor is full of hate and bugs.